addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1275,8 +1275,8 @@ export class ListRenderer extends Component {
         const groupIndex = group.list.records.indexOf(record);
         const isLastOfGroup = groupIndex === group.list.records.length - 1;
         const isDirty = record.isDirty || this.lastIsDirty;
-        const isEnterBehavior = hotkey === "enter" && (!record.canBeAbandoned || isDirty);
-        const isTabBehavior = hotkey === "tab" && !record.canBeAbandoned && isDirty;
+        const isEnterBehavior = hotkey === "enter" && (!record.isNew || isDirty);
```

Comment:
in the basic model, the `canBeAbandoned` logic was quite complicated. Don't you think it would be wiser to keep the function at least for now?